easybuild/easyconfigs/t/tbb/tbb-2021.9.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeNinja'
```

Comment:
While i like ninja, this does pull in quite a lot of extra deps for what typically is a pretty much nonexistent advantage the way we build software in easybuild (clean builds)

Reply:
Okay, I can easily revert to using `CMakeMake`.  I generally prefer Ninja for speed of build, but that's not really an issue that needs taking into account in this case.